plugins/eventmonitor/eventmonitorwidget.cpp

```diff
@@ -58,3 +74,8 @@ EventMonitorWidget::~EventMonitorWidget()
 {
     delete ui;
 }
+
+void EventMonitorWidget::pauseAndResume(bool pause)
+{
+    m_interface->setProperty("isPaused", pause);
```

Comment:
m_interface->setIsPaused(pause); should work too and avoids the error-prone string-based access